src/renderer/views/Playlist/Playlist.js

```diff
@@ -418,6 +423,20 @@ export default defineComponent({
       showToast(this.$t('User Playlists.SinglePlaylistView.Toast.This playlist does not exist'))
     },
 
+    showNoticesSometimes: function () {
+      if (this.alreadyShownNotice) return
```

Comment:
It's not a reactive property so I am not sure if this should be declared somewhere
@absidue ?

Reply:
Should I just declare it anyway?

Reply:
Just declare it, I doubt the performance matters too much here :)

Reply:
Done!

Reply:
Sorry missed the ping previously. Yes you should always declare variables somewhere, just assigning things to `this` is considered an anti-pattern with the Vue Options API.